src/rcheevos/trigger.c

```diff
@@ -176,6 +178,9 @@ int rc_evaluate_trigger(rc_trigger_t* self, rc_peek_t peek, void* ud, lua_State*
     char sub_primed = 0;
 
     do {
+      /* Clear the accumulator before each condset; matches behavior of values */
+      eval_state.accumulator_value.type = RC_VALUE_TYPE_UNSIGNED;
+      eval_state.accumulator_value.value.i32 = 0;
```

Comment:
This should be in `rc_test_condset_internal` where it's clearing out the `add_value`.

Reply:
At current time, keeping it where it is I should allow the non-pause pass to use the last remembered value from the pause pass.  This could have some logic-readability benefit.

Example:
https://retroachievements.org/game/29669
If I needed a pause in an achievement, I could set up the bulk of the very long pointer chain in the pause, then re-use it in the non-pause pass.  This is effectively a side-effect though, so perhaps avoiding a side effect is better than having some logic/readbility convenience.

Moving it to the internal test would effectively many pause pass and non-pause pass have a "different" recall accumulator, which is perhaps more consistent behavior.

Thoughts?

Reply:
I agree that sharing it between the pause and non-pause passes makes sense. I still think this is too far up. You've ended up putting this logic in three places where `rc_test_condset` is called. (There's a fourth in the unit tests that you missed). It should just be _in_ `rc_test_condset` (after the empty check).

Reply:
That makes send to me. I've moved it.